File: api-clients/dart/lib/model/study.dart
part of swagger.api;


@Entity()
class Study {
  /* Name of the study. */
  @Property(name: 'name')
  String name = null;
  
/* Year of the study. */
  @Property(name: 'year')
  int year = null;
  
/* Digital Object Identifier for the study. */
  @Property(name: 'doi')
  String doi = null;
  
/* PubMed ID for the study. */
  @Property(name: 'pubMedId')
  String pubMedId = null;
  
/* Slug based on the study name. */
  @Property(name: 'slug')
  String slug = null;
  
/* Key findings for the study. */
  @Property(name: 'keyFindings')
  String keyFindings = null;
  

  @Property(name: 'conditions')
  List<Condition> conditions = [];
  
/* Date and time record was created, UTC. */
  @Property(name: 'createdAt')
  DateTime createdAt = null;
  
/* Date and time record was updated, UTC. */
  @Property(name: 'updatedAt')
  DateTime updatedAt = null;
  
  Study();

  @override
  String toString()  {
    return 'Study[name=$name, year=$year, doi=$doi, pubMedId=$pubMedId, slug=$slug, keyFindings=$keyFindings, conditions=$conditions, createdAt=$createdAt, updatedAt=$updatedAt, ]';
  }

}

File: api-clients/async-scala/src/main/scala/io/swagger/client/model/Study.scala
package io.swagger.client.model

import org.joda.time.DateTime
import java.util.UUID


case class Study (
  name: String,  // Name of the study.
year: Option[Integer],  // Year of the study.
doi: Option[String],  // Digital Object Identifier for the study.
pubMedId: Option[String],  // PubMed ID for the study.
slug: Option[String],  // Slug based on the study name.
keyFindings: Option[String],  // Key findings for the study.
conditions: Option[List[Condition]],
createdAt: Option[DateTime],  // Date and time record was created, UTC.
updatedAt: Option[DateTime]  // Date and time record was updated, UTC.
)
